### ISSUE TITLE
After latest version update

### ISSUE THREAD

**Issue by shaileshck** [nVuln/homebridge-lg-thinq#79]
**Describe The Bug:**

**To Reproduce:**

**Expected behavior:**

**Logs:**

```
Show the Homebridge logs here (make sure homebridge DEBUG mode is turned `on`), remove any sensitive information.
```

**Plugin Config:**

```
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**: 

[12/09/2021, 12:21:18] [LGThinQ] Restoring existing accessory from cache: Washer Machine
[12/09/2021, 12:21:18] [LGThinQ] Restoring existing accessory from cache: Dishwasher
[12/09/2021, 12:21:18] [LGThinQ] START MQTT listener for thinq2 device
(node:1096) UnhandledPromiseRejectionWarning: Error: Request failed with status code 400
    at createError (/usr/local/lib/node_modules/homebridge-lg-thinq/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/usr/local/lib/node_modules/homebridge-lg-thinq/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/usr/local/lib/node_modules/homebridge-lg-thinq/node_modules/axios/lib/adapters/http.js:269:11)
    at IncomingMessage.emit (events.js:412:35)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1096) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:1096) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by nVuln** [comment]
can you try install `v1.2.0-mqtt.1` and test ?

**Comment by nVuln** [comment]
what is your country code ? from EU, right ?

**Comment by shaileshck** [comment]
India IN

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by shaileshck** [comment]
Currently on the same latest version as mentioned.
Country was showing USA which have changed to India and saved and restart the Config after update. But the issue continues

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by nVuln** [comment]
I already published test version `v1.2.0-mqtt.3`, please try install and see if it's worked

**Comment by shaileshck** [comment]
Yes the error has gone with the update.

Regards

Shailesh K.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
